apps/datafeeder/src/app/app.module.ts

```diff
@@ -16,7 +18,20 @@ import { UploadDataRulesComponent } from './presentation/components/upload-data-
     UploadDataComponent,
     UploadDataRulesComponent,
   ],
-  imports: [BrowserModule, AppRoutingModule, I18nModule, UiModule],
+  imports: [
+    BrowserModule,
+    AppRoutingModule,
+    I18nModule,
+    UiModule,
+    HttpClientModule,
+    TranslateModule.forRoot({
```

Comment:
Does it mean that 
```js
    TranslateModule.forRoot({
      loader: {
        provide: TranslateLoader,
        useFactory: HttpLoaderFactory,
        deps: [HttpClient],
      },
    }),
```
in `I18n` module is not enough ?

Reply:
Yeah, I will investigate this isssue

Reply:
Did you investigate ?
I think the translation is broken on search application, might be the cause